widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java

```diff
@@ -631,7 +631,7 @@ public void upgradeOfferChoiceSubmitSuccess(
                 Logger.d(TAG, "upgradeOfferChoiceSubmitSuccess");
                 if (submitter == MediaUpgradeOfferRepository.Submitter.CHAT) {
                     String requestedMediaType;
-                    if (offer.video != null) {
+                    if (offer.video != null && offer.video != MediaDirection.NONE) {
```

Comment:
what is this `MediaDirection.NONE` - aka when video offer can end up in this state?

Reply:
When we get an offer to update to the audio.